### PR TITLE
Add SystemRDL file type

### PR DESCRIPTION
**Pull request by RasmusGOlsen**
# New Features
Add SystemRDL source file type for solving feature request #47

### CODE REVIEW

File: pyEDAA/ProjectModel/__init__.py
Comment (Paebbels) [inline pyEDAA/ProjectModel/__init__.py:486]:
```suggestion
	def SystemRDLVersion(self) -> SystemRDLVersion:
```

File: pyEDAA/ProjectModel/__init__.py
Comment (Paebbels) [inline pyEDAA/ProjectModel/__init__.py:491]: This is not yet implemented on the fileset (and design) classes.

File: pyEDAA/ProjectModel/__init__.py
Comment (Paebbels) [inline pyEDAA/ProjectModel/__init__.py:497]:
```suggestion
	def SystemRDLVersion(self, value: SystemRDLVersion) -> None:
		self._srdlVersion= value
```

File: pyEDAA/ProjectModel/__init__.py
Comment (Paebbels) [inline pyEDAA/ProjectModel/__init__.py:495]:
```suggestion
	@SystemRDLVersion.setter
```

File: pyEDAA/ProjectModel/pySRDLModel.py
Comment (Paebbels) [inline pyEDAA/ProjectModel/pySRDLModel.py:1]: This file should come from a standalone PyPI package as suggested in issue #47.

**Review by Paebbels — CHANGES_REQUESTED** [review]

File: pyEDAA/ProjectModel/__init__.py
Comment (RasmusGOlsen) [reply pyEDAA/ProjectModel/__init__.py:490]: I have now implemented it in the Design and FileSet classes. I have deleted the pySRDLModel Python module. And imported the SystemRDLVersion from the pySRDLModel Python package. This package doesn't exist yet and has to be created and released to PyPi before this pull request can be merged.

File: pyEDAA/ProjectModel/__init__.py
Comment (Paebbels) [inline pyEDAA/ProjectModel/__init__.py:47]: Are we going to use `pySRDLModel` or `pySystemRDLModel`?

Normally I try to avoid abbreviations, OTOH pySVModel is also shortened because we got problems with headline and logos...

Any recommendation? Is SRDL also a known abbreviation like SV for SystemVerilog?

File: pyEDAA/ProjectModel/__init__.py
Comment (Paebbels) [reply pyEDAA/ProjectModel/__init__.py:490]: A repository is prepared. See #47.

File: pyEDAA/ProjectModel/__init__.py
Comment (RasmusGOlsen) [reply pyEDAA/ProjectModel/__init__.py:47]: I can see you went with `pySystemRDLModel` and I think that is fine.

Comment (Paebbels) [reply pyEDAA/ProjectModel/__init__.py:47]: Please take your decision / suggestion without my created repository. That can be refactored :).

Comment (RasmusGOlsen) [reply pyEDAA/ProjectModel/__init__.py:47]:
```suggestion
from pySystemRDLModel          import SystemRDLVersion
```

**Comment by Paebbels** [comment]
@RasmusGOlsen looks like this PR needs a merge or rebase to get merged. I tried to fix it online, but looks like I have no rights to do so.

Maybe you unchecked the checkbox that the maintainer can push to your branch to resolve conflicts.  
(I never needed that feature before, but I get asked this question myself when I create PRs at other places.)

**Comment by Paebbels** [comment]
Oh, and a reference in `requirements.txt` is needed.

**Comment by RasmusGOlsen** [comment]
> @RasmusGOlsen looks like this PR needs a merge or rebase to get merged. I tried to fix it online, but looks like I have no rights to do so.
> 
> Maybe you unchecked the checkbox that the maintainer can push to your branch to resolve conflicts. (I never needed that feature before, but I get asked this question myself when I create PRs at other places.)

I think it should be checked.

**Comment by RasmusGOlsen** [comment]
> Oh, and a reference in `requirements.txt` is needed.

I now updated the `requirements.txt` to include `pySystemRDLModel`, but this package needs to be uploaded to PyPi before this pull request can be accepted.

**Comment by Paebbels** [comment]
I released v0.1.0 at pySystemRDLModel to PyPI, so we can go on here.

File: requirements.txt
Comment (Paebbels) [inline requirements.txt:4]: Rerun pipeline by accepting this change :).
```suggestion
pySystemRDLModel >= 0.1.0
```

**Pull request merged**
